bin.js

```diff
@@ -60,6 +60,26 @@ if (argv[0] == 'server') {
   config.keys = keys
   var server = createSbot(config)
 
+  server.on('rpc:connect', function (rpc, isClient) {
+    if (rpc.id == '@' + keys.public) return
+
+    pull(
+      server.friends.hopStream(),
```

Comment:
do we have to worry about dangling streams of these?

will they emit values after the connection has been handled?